src/fixes/unordered_key.rs

```diff
@@ -68,7 +68,7 @@ impl Fix for UnorderedKeyFixer {
                 }
 
                 if line.is_empty()
-                    || lines.len() == line.number // Is this the last line?
+                    || lines.len() == i + 1 // Is this the last line?
```

Comment:
Could you tell me more about this change?
What's wrong with `line.number`?

Reply:
Well, for the same reason as [here](https://github.com/dotenv-linter/dotenv-linter/pull/450#discussion_r757765209)